dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/artifactbrowser/SendItemRequestAction.java

```diff
@@ -55,6 +56,7 @@
     protected HandleService handleService = HandleServiceFactory.getInstance().getHandleService();
     protected RequestItemService requestItemService = RequestItemServiceFactory.getInstance().getRequestItemService();
     protected BitstreamService bitstreamService = ContentServiceFactory.getInstance().getBitstreamService();
+    protected ItemService itemService = ContentServiceFactory.getInstance().getItemService();
```

Comment:
Unclear why itemService is being grabbed here (or why there's an `import org.dspace.content.service.ItemService;` above). It looks like it never gets used in this class, so there's no need to even pull it in.